fastapi/routing.py

```diff
@@ -42,6 +43,7 @@
 from pydantic import BaseModel
 from pydantic.error_wrappers import ErrorWrapper, ValidationError
 from pydantic.fields import ModelField, Undefined
+from pydantic.utils import lenient_issubclass as issubclass
```

Comment:
```suggestion
from pydantic.utils import lenient_issubclass
```

Could you please not replace the global `issubclass`?